include/net/nrf_cloud.h

```diff
@@ -535,7 +535,7 @@ struct nrf_cloud_init_param {
  *
  * @retval 0       If successful.
  * @retval -EACCES Already initialized or @ref nrf_cloud_uninit is in progress.
- *                 Otherwise, a (negative) error code is returned.
+ * @return A negative value indicates an error.
```

Comment:
Is it feasible to list the possible error values here?

Reply:
i don't think so, but you can look at the code... maybe you would disagree?

Reply:
I'm willing to take your word for it if you say it's not feasible